include/linux/soundwire/sdw_intel.h

```diff
@@ -4,15 +4,39 @@
 #ifndef __SDW_INTEL_H
 #define __SDW_INTEL_H
 
+/**
+ * struct sdw_intel_stream_params_data: configuration passed during
+ * the @params_stream callback, e.g. for interaction with DSP
+ * firmware.
```

Comment:
This patch is pretty good for me. I have only one idea.
In original file, there are descriptions for each filed in structure definition (like @xxxx zzzzzz).
to avoid any challenge, we should add these descriptions.

Reply:
yes, will add kernel-doc comments.